scripts/run-dev.sh

```diff
@@ -19,64 +19,9 @@ docker build . -f $VARIANT/Dockerfile -t $IMG
 docker rm -f $IMG || true
 docker run \
   -e IMPORTS=$IMPORTS \
-  -e WAV2LIP_MODEL_IDS="
```

Comment:
keep these around? easy to test

Reply:
yes, will revert this one. i did have to add `WIHSPER_TOKENIZER_FROM` for this model to work. we don't add that for any other whisper model